ocfweb/static/scss/pages/docs/_servers.scss

```diff
@@ -69,6 +69,13 @@
                     }
                 }
             }
+
+            .server-anchor {
+                display: block;
+                position: relative;
+                top: -75px;
```

Comment:
not sure either, i imagine you just want the same thing regular docs does?

https://github.com/ocf/ocfweb/blob/74c5e192d126bb73ba79eef81f3012608328588e/ocfweb/static/scss/pages/_docs.scss#L20-L27